new-lamassu-admin/src/pages/LoyaltyPanel/CouponCodes.js

```diff
@@ -0,0 +1,138 @@
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import { makeStyles, Box } from '@material-ui/core'
+import gql from 'graphql-tag'
+import * as R from 'ramda'
+import React, { useState } from 'react'
+
+import { Link, Button, IconButton } from 'src/components/buttons'
+import TitleSection from 'src/components/layout/TitleSection'
+import DataTable from 'src/components/tables/DataTable'
+import { H2, TL1 } from 'src/components/typography'
+import { ReactComponent as DeleteIcon } from 'src/styling/icons/action/delete/enabled.svg'
+
+import styles from './CouponCodes.styles'
+import CouponCodesModal from './CouponCodesModal'
+
+const useStyles = makeStyles(styles)
+
+const GET_COUPONS = gql`
+  query coupons {
+    coupons {
+      id
+      code
+      discount
+      soft_deleted
```

Comment:
I don't think we need to worry about softDeleted anywhere on the front-end. Can't we just ignore on the back-end queries?